docs/guide/install.md

```diff
@@ -53,6 +40,19 @@ Then run this command to move the `starport` executable to `/usr/local/bin/`:
 sudo mv starport /usr/local/bin/
 ```
 
+## Upgrading Your Starport Installation
+
+Before you install a new version of Starport, remove all existing Starport installations. 
+
+To remove the current Starport installation:
+
+1. On your terminal window, press `Ctrl+C` to stop the chain that you started with `starport chain serve`.
+1. Remove the Starport binary with `rm $(which starport)`.
+   Depending on your user permissions, run the command with or without `sudo`.
+1. Repeat this step until all `starport` installations are removed from your system.
```

Comment:
What does repeat this step refer to? Why does it have to be repeated?

Reply:
You may have Starport installed in `/usr/local/bin` and `~/go/bin`. Depending on your PATH, you may install using `curl | bash`, but still get an "old" binary from `~/go/bin`, for example.